### ISSUE TITLE
Oxford Neuroscience Open Science Co-ordinator: Changing Culture

### ISSUE THREAD

**Issue by cassgvp** [open-life-science/ols-1#15]
Project Lead: @cassgvp

Mentor: @npscience

Welcome to OLS-1! This issue will be used to track your project and progress during the program. Please use this checklist over the next few weeks as you start Open Life Science program :tada:.

***

Before Week 1 (Jan 20): Meet your mentor!
- [x] Create an account on [GitHub](https://github.com)
- [x] Complete **your own copy** of the [open leadership self-assessment](https://docs.google.com/document/d/1oQgdfj4lPnypAyb9_Ba0Zt7E8J5L6qMvuKwu0wgQsjs/edit?usp=sharing) and share it to your mentor
  
  If you're a group, each teammate should complete this assessment individually. This is here to help you set your own personal goals during the program. No need to share your results, but be ready to share your thoughts with your mentor.

- [x] Make sure you know when and how you'll be meeting with your mentor.

Before Week 2 (Jan 27): Cohort Call (Welcome to Open Life Science!)
- [x] Create an issue on the [OLS-1 GitHub repository](https://github.com/open-life-science/ols-1/issues/new) for your OLS work and share the link to your mentor.
- [x] Draft a brief vision statement using your goals

  [This lesson](https://mozilla.github.io/open-leadership-training-series/articles/introduction-to-open-leadership/stating-your-project-vision/) from the Open Leadership Training Series (OLTS) might be helpful

- [x] Leave a comment on this issue with your draft vision statement & be ready to share this on the call
- [x] Check the [Syllabus](https://openlifesci.org/ols-1) for notes and connection info for all the cohort calls.

Before Week 3 (Feb 3): Meet your mentor!
- [x] Look up two other projects and comment on their issues with feedback on their vision statement
- [x] Complete this [compare and contrast assignment](https://docs.google.com/document/d/1ukvqDRIYfvCapVMdE5hWP-0MkLNJ9T65X43O7F336Ac/edit?usp=sharing) about current and desired community interactions and value exchanges
- [x] Complete your Open Canvas ([instructions](https://mozilla.github.io/open-leadership-training-series/articles/opening-your-project/develop-an-open-project-strategy-with-open-canvas/), [canvas](https://docs.google.com/presentation/d/1MeJo0TyuMg_waLk1J4q9y1aAqKNMuRBlnmxEChSz-cQ/edit?usp=sharing))
- [x] Share a link to your Open Canvas in your GitHub issue
- [ ] Start your [Roadmap](https://mozilla.github.io/open-leadership-training-series/articles/opening-your-project/start-your-project-roadmap/)
- [ ] Comment on your issue with your draft Roadmap
- [x] Suggest a cohort name at the bottom of the shared notes and vote on your favorite with a +1

Before Week 4 (Feb 10): Cohort Call (Tooling and roadmapping for Open projects)
- [ ] Look up two other projects and comment on their issues with feedback on their open canvas.

Week 5 and more
- [ ] Create a GitHub repository for your project
- [ ] Add the link to your repository in your issue
- [ ] Use your canvas to [start writing a `README.md` file](https://mozilla.github.io/open-leadership-training-series/articles/opening-your-project/write-a-great-project-readme/), or landing page, for your project
- [ ] Link to your README in a comment on this issue
- [ ] Add an [open license](https://mozilla.github.io/open-leadership-training-series/articles/get-your-project-online/sharing-your-work-in-the-open/) to your repository as a file called `LICENSE.md`
- [ ] Add a [Code of Conduct](https://mozilla.github.io/open-leadership-training-series/articles/building-communities-of-contributors/write-a-code-of-conduct/) to your repository as a file called `CODE_OF_CONDUCT.md`
- [ ] Invite new contributors to into your work!

This issue is here to help you keep track of work as you start Open Life Science program. Please refer to the [OLS-1 Syllabus](https://openlifesci.org/ols-1) for more detailed weekly notes and assignments past week 4.

**Comment by cassgvp** [comment]
I’m working with 225 research groups across Oxford Neuroscience to create a community of open and inclusive researchers who will develop the policies, governance and structures we would like in place to support us in practicing open science.

I’m working open because I believe this is the most effective way to deliver justice oriented science.

**Comment by ChiaraBertipaglia** [comment]
Your project sounds so interesting, I think we are moved by the same vision! I'd love to see how you're mapping stakeholders to obtain necessary buy-in (or perhaps you are already past that point?)

It would be great if you had time to take a look at my roadmap, so we could exchange ideas. Thanks! https://github.com/open-life-science/ols-1/issues/9

It also looks like you might be involved in community management tasks? Do you know the community of community managers that Lou Woodley has put together https://www.cscce.org/? You are welcome to join our slack channel https://cscce-slack-invite-bot.herokuapp.com/

**Comment by cassgvp** [comment]
Hi @ChiaraBertipaglia! Yes, I was thinking exactly the same! It would be great to stay close to each other though this program to share ideas/experience 🙌. Will definitely look at your roadmap! 

Many thanks for the links!

**Comment by cassgvp** [comment]
My open canvas: [Policies for establishing behaviour change toward the adoption of open science practices in OxN. Developed by the community, for the community.](https://docs.google.com/presentation/d/1OgutqAQL5KdmXzfGt3W6JyMqgPOtRiKSCUxvrlfLzew/edit?usp=sharing)

**Comment by cassgvp** [comment]
#11 #16 #6 #4 #9: Is this all of us doing work our establishing community? Did I miss anyone? Would you all be interested in making some kind of mechanisms for us to exchange ideas (in addition to the normal cohort calls)? I can't think of how just yet, but I'm sure we will be researching and generating resources we can all make use of!

**Comment by ChiaraBertipaglia** [comment]
Yes, I am super interested in this. Perhaps at this early stage we can take some time to look at each other's material and then we can set up some space to exchange thoughts cohesively? Ex.: could we create a channel in Gitter? Looking at other people's open canvases has already been helpful as it made me realize some blind spots in my thinking

**Comment by bonetcarne** [comment]
Yes, we should exchange info and tips and stay communicated. Maybe the easiest would be to meet in a room during the next group call next week?

**Comment by Karvovskaya** [comment]
Hi @cassgvp ! I am super interested in your project. It would be great to share experience and support each other. I am very curious about how to get started. Do I understand it right that there is already some commitment from the researchers to be part of the community?

**Comment by cassgvp** [comment]
> Hi @cassgvp ! I am super interested in your project. It would be great to share experience and support each other. I am very curious about how to get started. Do I understand it right that there is already some commitment from the researchers to be part of the community?

There are some/many people who are already doing the work "for free" (i.e. in addition to their research load), so in that way they are already committed. What is missing is a formal "community" so they can communicate with each other, have their views represented in management meetings, and receive recognition for their work.

**Comment by SamGuay** [comment]
Hi Cass! 

Feels great to see another OHBM fellow in the cohort! I love your mission, and I'm eager to spy on your roadmad :eyes:. 

> Would you all be interested in making some kind of mechanisms for us to exchange ideas (in addition to the normal cohort calls)? I can't think of how just yet, but I'm sure we will be researching and generating resources we can all make use of!

For our future platform, Zulip (an open source alternative to Slack) sponsored us a free hosted instance where we can chat, have conference call, etc. We haven't publicized it yet, but feel free to join :arrow_right: [chat.openscience.ca](https://chat.openscience.ca). 

Other option could be having a private chat on Gitter? But we might loose some useful discussion since it's a single linear thread.

As of now, it's hard to follow the multiple convos we have throughout all the issues :dizzy_face:

**Comment by billbrod** [comment]
Your Open Canvas looks great! Looks like you have a good idea of what you want to create and what you need to get there. From your canvas, it's not clear to me if/how your users differ from your contributors; are they a subset of the contributors (only people at Oxford and not the research and innovation committee), or are they a different group of people within that category (as in, your users are a different bunch of engaged ECRs and PIs than those that are your contributors)? Or are are they the same people?

Building a culture of open science is so important (and difficult! Everywhere I've been, it's been very piecemeal, with a handful of labs and people interested in it, but nothing connecting them) -- I'm excited to watch all the groups that want to do this moving forward!

**Comment by cassgvp** [comment]
> Hi Cass!
> 
> Feels great to see another OHBM fellow in the cohort! I love your mission, and I'm eager to spy on your roadmad 👀.
> 
> > Would you all be interested in making some kind of mechanisms for us to exchange ideas (in addition to the normal cohort calls)? I can't think of how just yet, but I'm sure we will be researching and generating resources we can all make use of!
> 
> For our future platform, Zulip (an open source alternative to Slack) sponsored us a free hosted instance where we can chat, have conference call, etc. We haven't publicized it yet, but feel free to join ➡️ [chat.openscience.ca](https://chat.openscience.ca).
> 
> Other option could be having a private chat on Gitter? But we might loose some useful discussion since it's a single linear thread.
> 
> As of now, it's hard to follow the multiple convos we have throughout all the issues 😵

Totally agree on getting lost in the threads, and very happy to join you on Zulip! The link didn't take me through to what I expected!?! Is it running yet?

I totally recognise you from OHBM now! Rome was pretty hectic for me, and this year is going to be even busier! Are you involved in any of the local organising committee stuff? What to help out in the Open Science Room?! 😁

**Comment by cassgvp** [comment]
> Your Open Canvas looks great! Looks like you have a good idea of what you want to create and what you need to get there. From your canvas, it's not clear to me if/how your users differ from your contributors; are they a subset of the contributors (only people at Oxford and not the research and innovation committee), or are they a different group of people within that category (as in, your users are a different bunch of engaged ECRs and PIs than those that are your contributors)? Or are are they the same people?
> 
> Building a culture of open science is so important (and difficult! Everywhere I've been, it's been very piecemeal, with a handful of labs and people interested in it, but nothing connecting them) -- I'm excited to watch all the groups that want to do this moving forward!

Thank you for the feedback! I think I was struggling a bit to understand if/how my users are different from contributors. I'm probably still not sure! I do want the contributors to be users, and the users to be contributors... Still a lot to work out it seems! Thank you for highlighting!

**Comment by cassgvp** [comment]
My [compare and contrast assignment](https://docs.google.com/document/d/1AjQY0yU_FoniluHlrhDBOpUXLDWly-wBMrcjp4pJiMU/edit?usp=sharing) about current and desired community interactions and value exchanges.

**Comment by malvikasharan** [comment]
Hi Project Lead,

This is your project report file: 
https://hackmd.io/DPcjYrUIRBuHFYfwEddfIA
Please start working on it for the final presentation. We will send more info in the weekly email.  

Best,
 OLS team
